mockintosh/__init__.py

```diff
@@ -134,6 +134,9 @@ def _handle_cli_args_logging(args: list, fmt: str) -> None:
         logging.basicConfig(level=logging.INFO, format=fmt)
         logging.getLogger('pika').setLevel(logging.CRITICAL)
         logging.getLogger('rsmq').setLevel(logging.CRITICAL)
+    logging.getLogger('botocore').setLevel(logging.CRITICAL)
```

Comment:
The indentation looks strange here. It either needs empty line to separate the block from above `if`, or fixed intent to fit into `else`

Reply:
@undera the indentation is intentional. Those packages are too noisy on `DEBUG` level and even on `INFO` level. So I've basically disabled almost all logging that comes from those packages. I can re-enable them if you would like to.